src/index.ts

```diff
@@ -26,10 +50,23 @@ const Vessel = (
   } = {
     baseUrl: BASE_URL,
   }
-) => {
+): VesselClient => {
+  if (isSSR()) {
+    return {
```

Comment:
Just confirming my understand of SSR - basically after this code is sent to the client, the component will remount and call this function again but ssr will be true now and this will all execute as expected?

Reply:
Chat GPT says yes 🙃